#! /usr/bin/env python

PACKAGE='jsk_perception'

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("bin_size", int_t, 0, "", 30, 1, 100)
gen.add("pixel_min_value", int_t, 0, "", 0, 0, 255)
gen.add("pixel_max_value", int_t, 0, "", 180, 0, 255)
gen.add("histogram_top_n_ratio", double_t, 0, "", 0.5, 0.0, 1.0)
exit(gen.generate(PACKAGE, "jsk_perception", "TabletopColorDifferenceLikelihood"))
